internal/lib/lib.go

```diff
@@ -995,10 +995,6 @@ func FetchSEGroupWithMarkerSet(client *clients.AviClient, overrideUri ...NextPag
 		if aviError, ok := err.(session.AviError); ok && aviError.HttpStatusCode == 403 {
 			//SE in provider context no read access
 			utils.AviLog.Debugf("Switching to admin context from  %s", GetTenant())
-			SetAdminTenant := session.SetTenant(GetAdminTenant())
-			SetTenant := session.SetTenant(GetTenant())
-			SetAdminTenant(client.AviSession)
-			defer SetTenant(client.AviSession)
```

Comment:
code to fallback client to admin tenant needs to be added here?

Reply:
this function is called when Advanced_L4 is set (wcp mode), there no tenant is defined in the config map which means the admin tenant client will be used by default